files/background.js

```diff
@@ -44,6 +44,20 @@ chrome.runtime.onInstalled.addListener(function() {
                 });
 
         });
+
+	if (details.url.indexOf("en.wiktionary.org/wiki") < 0) return;
+	if (details.url.indexOf("en.wiktionary.org/wiki/Appendix:") >= 0) return;
+
+	getSavedItem(wiktionaryFilterPrune, (prune) => {
+	    if (!prune) return;
+	    getSavedItem(wiktionaryFilterPruneExcept, (pruneExcept) => {
+		let pruneExceptList = pruneExcept.split(', ');
```

Comment:
The split needs to be better about spacing; try this:

```suggestion
let pruneExceptList 
              = pruneExcept
                .split(',')
                .map((s) => s.trim());
```

I was able to break the pruning by putting in `Foo,Bar`.